docs/connect.rst

```diff
@@ -49,4 +49,32 @@ Configuration Options
 
   * Type: int
   * Default: 1000
-  * Importance: low
\ No newline at end of file
+  * Importance: low
+
+``key.subject.name.strategy``
+  Determines how to construct the subject name under which the key schema is registered with the schema registry. By default, <topic>-key is used as subject.
+
+  * Type: class
+  * Default: class io.confluent.kafka.serializers.subject.TopicNameStrategy
```

Comment:
What other listing the other options?

Reply:
@rhauch I'm not sure I understand the comment. Can you please elaborate.